query/rds/rds_cluster_deletion_protection_enabled.sql

```diff
@@ -0,0 +1,16 @@
+ select
```

Comment:
`query/rds/rds_db_cluster_deletion_protection_enabled.sql`

We should be consistent with file names, most include `db`, e.g., `rds_db_cluster`, so this one should too